crates/cast/src/starknet_commands/account/delete.rs

```diff
@@ -0,0 +1,118 @@
+use anyhow::{anyhow, bail, Context, Result};
+use camino::Utf8PathBuf;
+use cast::helpers::response_structs::AccountDeleteResponse;
+use cast::helpers::scarb_utils::get_scarb_manifest;
+use clap::Args;
+use serde_json::Map;
+use serde_json::Value;
+use std::fs::{File, OpenOptions};
+use std::io;
+use std::io::Read;
+use std::io::Write;
+use std::str::Split;
+
+#[derive(Args, Debug)]
+#[command(about = "Delete an account information from the accounts file")]
+pub struct Delete {
+    /// Name of the account to be deployed
+    #[clap(short, long)]
+    pub name: Option<String>,
```

Comment:
should this be optional? This might lead to confusing situation, when you delete an account (specified by Scarb.toml) without removing it from Scarm.toml itself. I'd suggest we require users provided the name.

Especially so that I see you've marked it as required in the docs :+1:

Reply:
i think we must leave as default behavior to delete the profile as you suggest. and optionally a --no_delete_profile flag. are you ok?

Reply:
Maybe let's make it an explicit `--delete-profile false` flag and set it to true as default (if not passed).

Reply:
ok, then this is the behavior:
--delete-profile false : to not delete profile in scarb
--delete-profile true : to delete profile in scarb
ommited --delete-profile:  to delete profile in scarb